code/modules/reagents/reagents/medical.dm

```diff
@@ -288,7 +288,7 @@
 	description = "Dylovene is a broad-spectrum antitoxin."
 	color = "#A8F59C"
 	scannable = TRUE
-	purge_list = list(/datum/reagent/toxin, /datum/reagent/toxin/xeno_neurotoxin, /datum/reagent/consumable/drink/coffee)
+	purge_list = list(/datum/reagent/toxin, /datum/reagent/toxin/xeno_neurotoxin, /datum/reagent/consumable/drink/atomiccoffee, /datum/reagent/consumable/larvajelly)
```

Comment:
Undocumented changes.

Reply:
Here's the documentation~

Upped the price on cases of pure chems, to lower the chances of people just vending them then making 20 injectors anyway. Still doable, but it'll be most of your loadout.